src/robot/running/arguments/argumentparser.py

```diff
@@ -23,10 +23,17 @@
 if PY2:
     from inspect import getargspec, ismethod
 
+
     def getfullargspec(func):
-        return getargspec(func) + ([], None, {})
+        return getargspec(unwrap(func)) + ([], None, {})
+
+
+    def unwrap(func):
+        while hasattr(func, '__wrapped__'):
+            func = func.__wrapped__
+        return func
```

Comment:
I'm not sure is this a good idea. In my opinion we should unwrap keywords only when using Python 3 where it's supported by Python itself. If we were to support Python 2, we needed to add some tests for it.

We do need `unwrap` implementation here, though, because it's used by `PythonArgumentParser` below. It's implementation can just be:

```python
def unwrap(func):
    return func
```